memo/managers.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+from django.db import models
+from django.db.models import Count, Q, FloatField, Avg, Sum, Max
+from django.db.models.functions import Cast
+
+
+class CardManager(models.Manager):
+    def get_filter_by_deck(self, deck):
+        return (
+            super(CardManager, self)
+            .get_queryset()
+            .filter(deck_id=deck, card_answers__isnull=False)
+        )
+
+    def get_total_answer(self, deck):
+        return (
+            self.get_filter_by_deck(deck)
+            .annotate(total_answers=Count("card_answers"))
+            .aggregate(Sum("total_answers"))
+            .get("total_answers__sum")
+        )
+
+    def get_last_response(self, deck):
+        last_response = (
+            self.get_filter_by_deck(deck)
+            .annotate(last_response=Max("card_answers__updated_at"))
+            .aggregate(Max("last_response"))
+            .get("last_response__max")
+        )
+
+        if last_response != None:
```

Comment:
Python will automatically check if the value if truthy or falsy if you put it like this =]. More about it [here](https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/)
```suggestion
        if last_response:
```

Reply:
> init__.py

No, it's not happening to my project. It's not even creating or using this file in the `tests/` folder structure:
![image](https://user-images.githubusercontent.com/42613997/182442771-a67df27c-f178-4a20-bc8d-ab8c8cb28392.png)
Does pytest need to create that file? 

Thank you for commenting about this line and linking the article about true/false. It makes sense since all other (or almost every) language has the same concept when the value is true or false. 🥲

Reply:
Oh I see, you are using pytest. I didn't realized, so I was using django for testing and it was complaning about this file. Alright then, carry on :sweat_smile:

Reply:
Oh, okay! Thank you for replying my last comment. I'm going to commit to the suggested updates, and I also have sent the invite to this project so you can approve the PR.